tsconfig.build.jobs.contact-retrieve-complete.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.base.json",
+  "files": [],
+  "references": [
+    { "path": "packages/ssm-cache" },
+    { "path": "packages/s3-client" },
+    { "path": "packages/twilio-client" },
+    { "path": "jobs/contact-retrieve-recording-url" }
```

Comment:
Is this reference correct?

Reply:
Oops. Good catch! Switched those around. Fixed.